public/index.html

```diff
@@ -107,58 +107,41 @@
-          if (isElectron) {
-            segmentWriteKey = '%REACT_APP_SEGMENT_ELECTRON%'
-          }
-          !function(){var analytics=window.analytics=window.analytics||[];if(!analytics.initialize)if(analytics.invoked)window.console&&console.error&&console.error("Segment snippet included twice.");else{analytics.invoked=!0;analytics.methods=["trackSubmit","trackClick","trackLink","trackForm","pageview","identify","reset","group","track","ready","alias","debug","page","once","off","on"];analytics.factory=function(t){return function(){var e=Array.prototype.slice.call(arguments);e.unshift(t);analytics.push(e);return analytics}};for(var t=0;t<analytics.methods.length;t++){var e=analytics.methods[t];analytics[e]=analytics.factory(e)}analytics.load=function(t,e){var n=document.createElement("script");n.type="text/javascript";n.async=!0;n.src="https://analytics.audius.co/"+t;var a=document.getElementsByTagName("script")[0];a.parentNode.insertBefore(n,a);analytics._loadOptions=e};analytics.SNIPPET_VERSION="4.1.0";
-            analytics.load(segmentWriteKey)
-          }}();
-
-          if (window.analytics) {
-            const SESSION_START = 'Session Start'
-            let source
-            if (isNativeMobile == 'true') {
-              if (/android/i.test(userAgent)) source = 'Android App'
-              else if (/iPad|iPhone|iPod/.test(userAgent) && !window.MSStream) source = 'iOS App'
-              else source = 'Unknown App'
-            } else if (isMobileCheck) {
-              if (/windows phone/i.test(userAgent)) source = 'Windows Phone Web'
-              else if (/android/i.test(userAgent)) source = 'Android Web'
-              else if (/iPad|iPhone|iPod/.test(userAgent) && !window.MSStream) source = 'iOS Web'
-              else source = 'Unknown Mobile Web'
-            } else if (isElectron) {
-              const platform = window.navigator.platform
-              const macosPlatforms = ['Macintosh', 'MacIntel', 'MacPPC', 'Mac68K']
-              const windowsPlatforms = ['Win32', 'Win64', 'Windows', 'WinCE']
-
-              if (macosPlatforms.indexOf(platform) !== -1) source = 'MacOS'
-              else if (windowsPlatforms.indexOf(platform) !== -1) source = 'Windows'
-              else if (/Linux/.test(platform)) source = 'Linux'
-              else source = 'Unknown Desktop'
-            } else {
-              source = 'Web'
-            }
-
-            window.analytics.track(SESSION_START, { source })
+    <!-- start Adroll -->
+    <script type="text/javascript">
+      const adroll_adv_id = '%REACT_APP_ADROLL_AVD_ID%';
+      const adroll_pix_id = '%REACT_APP_ADROLL_PIX_ID%';
+      const adroll_version = "2.0";
+      console.log({
+        adroll_adv_id,
+        adroll_pix_id
+      })
+      // if (nodeEnv === 'production' && reactEnv === 'production' && !adroll_adv_id.includes('%') && !adroll_pix_id.includes('%')) {
```

Comment:
Is this comment part of Adroll? weird lol

Reply:
no it's something we add to only init an analytics library if we're in production

Reply:
I need to uncomment it out, thx